cypress-custom/integration/fee.test.ts

```diff
@@ -54,9 +54,53 @@ describe('Fetch and persist fee', () => {
 
   // TODO: not sure if it's easy to test this
   it('Re-fetched when it expires', () => {
+    const NETWORK = ChainId.RINKEBY.toString()
+    const TOKEN = 'ETH'
+    const KEY = 'redux_localstorage_simple_fee'
+    const VALUE = JSON.stringify({
+      [NETWORK]: {
+        [TOKEN]: {
+          fee: {
+            minimalFee: '777',
+            feeRatio: 777,
+            expirationDate: new Date(Date.now() - 200000).toISOString()
```

Comment:
my initial thinking was to set some time in the future always and advance time. Maybe this helps https://docs.cypress.io/api/commands/tick.html#Syntax

Reply:
Reason is, we want to really reproduce a workflow where the user has a key, come back some days later, and then we make sure he uses the good fee

Reply:
Hmm could be a good idea, didn't see tick before. Will check out tomorrow !

Reply:
Moved to an issue so we can merge this PR https://github.com/gnosis/gp-swap-ui/issues/125